packages/flutter_tools/lib/src/android/android_builder.dart

```diff
@@ -6,11 +6,11 @@ import 'dart:async';
 
```

Comment:
Seems like `AndroidBuilder` is mostly just a trivial wrapper around `buildGradleAar` and `buildGradleApp` now. Can we remove the entire file? (Maybe in a follow-up PR.)

Reply:
+1. I filed https://github.com/flutter/flutter/issues/43863. Originally this was needed to test the build commands because the current build process is messy and requires mocking too many files/process calls, etc.